Tests/configure_and_test_integration_instances.py

```diff
@@ -54,6 +58,91 @@ def determine_server_url(ami_env):
     return server_url
 
 
+def get_server_numeric_version(ami_env, prints_manager):
+    '''
+    Gets the current server version
+    Arguments:
+        ami_env: (str)
+            AMI version name.
+        prints_manager: (ParallelPrintsManager)
+            Print manager object.
+    Returns:
+        (str) Server numeric version
+    '''
+    images_file_name = './Tests/images_data.txt'
+    if not os.path.isfile(images_file_name):
+        return '99.99.98'  # latest
+    try:
+        with open(images_file_name, 'r') as image_data_file:
+            image_data = [line for line in image_data_file if line.startswith(ami_env)]
+            if len(image_data) != 1:
+                prints_manager.add_print_job('Did not get one image data for server version, got {}'.format(image_data),
```

Comment:
either you need print manager with variable thread_index or you can simply use print